clang/test/Sema/assertion-crash.cpp

```diff
@@ -0,0 +1,19 @@
+// RUN: %clang_cc1 -fsyntax-only -std=c++11 %s -verify
```

Comment:
Normally if we add a new test just for a specific bug report we name it something like `GH35603.cpp` so we can readily identify the test is linked to github issue 35603. If we add the test to an existing test file we wrap the test in namespace of the same name.